python/paddle/optimizer/adamw.py

```diff
@@ -163,6 +165,9 @@ def __init__(self,
         self._apply_decay_param_fun = apply_decay_param_fun
         self._coeff = coeff
         self._lr_to_coeff = dict()
+        if lr_ratio is not None:
+            assert isinstance(lr_ratio, Callable)
+        self._lr_ratio = lr_ratio
```

Comment:
you should think about how many kernel will be affected by  "lr_ratio". 
if you only want the lr_ratio the affect gpu and cpu kernel, you should raise an Unimplement Error for xpu and npu here.

Reply:
done.